demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -103,13 +104,40 @@ def is_valid_file(self, validate_rn: bool = True, skip_test_conf: bool = False)
 
     def is_valid_as_deprecated(self):
         """Check if the integration is valid as a deprecated integration."""
-
         answers = [
-            self.is_valid_deprecated_integration_display_name(),
-            self.is_valid_deprecated_integration_description(),
+            self._is_valid_deprecated_integration_display_name(),
+            self._is_valid_deprecated_integration_description(),
         ]
         return all(answers)
 
+    def _is_valid_deprecated_integration_display_name(self) -> bool:
+        is_valid = True
+        is_deprecated = self.current_file.get('deprecated', False)
+        display_name = self.current_file.get('display', '')
+        if is_deprecated:
+            if not display_name.endswith('(Deprecated)'):
+                error_message, error_code = Errors.invalid_deprecated_integration_display_name()
+                if self.handle_error(error_message, error_code, file_path=self.file_path):
+                    is_valid = False
+        return is_valid
+
+    def _is_valid_deprecated_integration_description(self) -> bool:
+        is_valid = True
+        is_deprecated = self.current_file.get('deprecated', False)
+        description = self.current_file.get('description', '')
+        deprecated_v2_regex = r'Deprecated\. Use the .+ integration instead\.'
```

Comment:
Small remark. I would build the code so it iterates via a list of regexes instead of if/elif/else. That way it is easier in the future to add additional regex patterns as we see fit.